cpp/src/slice2swift/Gen.cpp

```diff
@@ -132,18 +132,10 @@ Gen::ImportVisitor::visitModuleStart(const ModulePtr& p)
 bool
 Gen::ImportVisitor::visitClassDefStart(const ClassDefPtr& p)
 {
-    //
     // Add imports required for base class
-    //
-    ClassDefPtr base = p->base();
-    if (base)
-    {
-        addImport(dynamic_pointer_cast<Contained>(base), p);
```

Comment:
It's fine to pass a `nullptr` to `addImport`.
So checking `base` before calling it is unnecessary.

Also no need to cast to `Contained` now. `ClassDef` already inherits from `Contained`.